### PR TITLE
Update roadmap.md

### PR DESCRIPTION
**Pull request by magowan**
Bringing roadmap.md into line with current roadmap and processes .

### CODE REVIEW

**Comment by magowan** [comment]
Updates based on discussion in incubation working group so we can address any glaring inconsistencies within our docs and then improve on this later. (With Use Case details for example)

**Review by ariel-adam — APPROVED** [review]
Looks good, we can later on extend the sections on the use cases

File: roadmap.md [hunk 1]
```diff
@@ -1,87 +1,27 @@
...
 
-# Short-Term Roadmap
-The short-term roadmap aims to achieve the following:
-- MVP stack for running confidential containers
-- Based on and compatible with Kata Containers 2
-- Based on at least one confidential computing implementation (SEV, TDX, SE, etc)
-- Integration with Kubernetes: kubectl apply -f confidential-pod.yaml
-
-The work is targeted to be completed by end of November 2021 and includes 3 milestones:
-![September 2021](./images/RoadmapSept2021.jpg)
```
Comment (mythi) [inline roadmap.md:19]: the images could be `git rm`d

File: roadmap.md [hunk 2]
```diff
@@ -1,87 +1,27 @@
...
 
-# Longer-Term Roadmap
+In CoCo use case driven development we identify the main functional requirements the community requires by focusing on key use cases.
 
-Focused meetings will be set up to discuss architecture and the priority of longer-term objectives 
-in the process of being set up.
+This helps the community deliver releases which address real use cases customers require and focusing on the right priorities. The use case driven development approach also includes developing the relevant CI/CDs to ensure end-to-end use cases the community delivers work over time.
 
-Each meeting will have an agreed focus with people sharing material/thoughts ahead of time.
+We target the following use cases"
```
Comment (mythi) [inline roadmap.md:19]:
```suggestion
We target the following use cases:
```

File: roadmap.md [hunk 3]
```diff
@@ -1,87 +1,27 @@
...
-![November 2021](./images/RoadmapNov2021.jpg)
-- **November 2021**
-  - Image is optionally stored on an encrypted, ephemeral block device
-  - Image is decrypted with a key obtained from a key brokering service (KBS)
-  - Integration with kubelet
-
-For additional details on each milestone see [Confidential Containers v0](https://docs.google.com/presentation/d/1SIqLogbauLf6lG53cIBPMOFadRT23aXuTGC8q-Ernfw/edit#slide=id.p).
+When looking at the project's roadmap we distinguish between the short-term roadmap (2-6 months) vs. the mid/long-term roadmap (6-18 months):
+- The **short-term roadmap** is focused on achieving an end-to-end, easy to deploy and stable confidential containers solution. We track this work on a number of github boards (detailed later on)
+- The **mid/long-term solutions** focuses on use case driven development (detailed later on)
```
Comment (mythi) [inline roadmap.md:5]:
```suggestion
- The [short-term roadmap](#short-term-roadmap) is focused on achieving an end-to-end, easy to deploy and stable confidential containers solution. We track this work on a number of github boards
- The [mid and long term roadmap](#mid-and-long-term-roadmap) focuses on use case driven development
```

File: roadmap.md [hunk 4]
```diff
@@ -1,87 +1,27 @@
...
-  technology
-- Develop a small, simple, secure, lightweight and high performance OCI container image 
-  management library [image-rs](https://github.com/confidential-containers/image-rs) for 
-  confidential containers.
-- Develop small, simple shim firmware ([td-shim](https://github.com/confidential-containers/td-shim))
-  in support of trusted execution environment for use with cloud native confidential containers.
-- Document threat model and trust model, what are we protecting, how are we achieving it.
-- Identify technical convergence points with other confidential computing projects both inside 
-  and outside CNCF.
+# Mid-Term Roadmap/ Long-Term Roadmap
```
Comment (mythi) [inline roadmap.md:13]: (to match with the anchor suggested above)
```suggestion
# Mid and Long Term Roadmap
```

**Review by mythi — COMMENTED** [review]
minor editorial suggestions

File: roadmap.md (hunk 1 above)
Comment (magowan) [reply roadmap.md:19]: Good point

Comment (magowan) [reply roadmap.md:19]: :-) I just updated PR with their removal ...

**Review by mythi — APPROVED** [review]
thanks @magowan

**Comment by ariel-adam** [comment]
Merging this work to converge on CoCo incubation submissions

**Pull request merged**